docs/contributing/building.rst

```diff
@@ -65,6 +67,8 @@ configured. Then, create and activate the development environment:
 The ``conda env update`` line can be run later to update your environment. Deactivate your environment
 ``conda deactivate``, then run the update commands, then reactivate ``conda activate katana-dev``.
 
+**NOTE** : If building katana-enterprise, please use [these](https://docs.k9h.dev/0.1.99.dev+3.8.04ffe8.b7efb9/contributing/index.html#conda) commands instead of the ones above. Once they've been run, the remaining process remains the same as below commands. 
```

Comment:
- avoid hyperlinks when we are linking internally
- There is a `.. note::` directive in RST.
- If we are going to go through the effort of referencing the enterprise addenda here, we might as well do this for all the addenda items rather than just the Conda one.